package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -551,46 +622,57 @@ def run(self, **kwargs):
                 self._update_water_selection()
 
             s1_frame_results_dict = {}
-            if self.selection1_type in ('donor', 'both') and self._water_acceptors:
+            if self.selection1_type in ('donor', 'both') and \
+            self._water_acceptors:
```

Comment:
Please indent for legibility. Follow [PEP8 indentation rules](https://www.python.org/dev/peps/pep-0008/#indentation)